lib/octobox/notifications/sync_subject.rb

```diff
@@ -28,7 +28,7 @@ def update_subject_in_foreground(force = false)
       private
 
       def download_subject
-        user.subject_client.get(subject_url)
+        github_client.get(subject_url)
```

Comment:
I believe this is the actual root issue I am attempting to fix. Before my changes `user.subject_client` would always use the `subject_token` if the GithubApp is installed.

After this change it will only use this token if the repository is accessible by the Github App

Reply:
👍